client/i18n/locales/german/motivation.json

```
@@ -1,819 +1,203 @@
       "author": "Thomas A. Edison"
     },
-    {
-      "quote": "Our greatest weakness lies in giving up. The most certain way to succeed is always to try just one more time.",
-      "author": "Thomas A. Edison"
-    },
-    {
-      "quote": "The harder the conflict, the more glorious the triumph.",
-      "author": "Thomas Paine"
-    },
-    {
-      "quote": "The Web as I envisaged it, we have not seen it yet. The future is still so much bigger than the past.",
-      "author": "Tim Berners-Lee"
-    },
-    {
-      "quote": "Failure is the condiment that gives success its flavor.",
-      "author": "Truman Capote"
-    },
-    {
-      "quote": "Those who says it cannot be done should not interrupt the person doing it.",
-      "author": "Unknown"
-    },
-    {
-      "quote": "Even if you fall on your face, you're still moving forward.",
-      "author": "Victor Kiam"
-    },
-    {
-      "quote": "It's not whether you get knocked down, it's whether you get up.",
-      "author": "Vince Lombardi"
-    },
-    {
-      "quote": "I dream my painting and I paint my dream.",
-      "author": "Vincent van Gogh"
-    },
-    {
-      "quote": "Let us cultivate our garden.",
-      "author": "Voltaire"
-    },
-    {
-      "quote": "Aim for the moon. If you miss, you may hit a star.",
-      "author": "W. Clement Stone"
-    },
-    {
-      "quote": "The way to get started is to quit talking and begin doing.",
-      "author": "Walt Disney"
-    },
-    {
-      "quote": "You miss 100% of the shots you don't take.",
-      "author": "Wayne Gretzky"
-    },
-    {
-      "quote": "Don't let yesterday take up too much of today.",
-      "author": "Will Rogers"
-    },
-    {
-      "quote": "Even if you're on the right track, you'll get run over if you just sit there.",
-      "author": "Will Rogers"
-    },
-    {
-      "quote": "Do not wait to strike till the iron is hot; but make it hot by striking.",
-      "author": "William Butler Yeats"
-    },
-    {
-      "quote": "You cannot swim for new horizons until you have courage to lose sight of the shore.",
-      "author": "William Faulkner"
-    },
-    {
-      "quote": "Be not afraid of greatness. Some are born great, some achieve greatness, and others have greatness thrust upon them.",
-      "author": "William Shakespeare"
-    },
-    {
-      "quote": "We know what we are, but not what we may be.",
-      "author": "William Shakespeare"
-    },
-    {
-      "quote": "In theory there is no difference between theory and practice. In practice there is.",
-      "author": "Yogi Berra"
-    },
-    {
-      "quote": "You can see a lot by just looking.",
-      "author": "Yogi Berra"
-    },
-    {
-      "quote": "There is no elevator to success, you have to take the stairs.",
-      "author": "Zig Ziglar"
-    },
-    {
-      "quote": "You don't have to be great to start, but you have to start to be great.",
-      "author": "Zig Ziglar"
-    }
   ]
-}
+}
```

same here, don't remove the last new line
```suggestion
}

```

---

@ieahleen Thank you for your advice and help!